pygmt/clib/session.py

```diff
@@ -1190,6 +1190,45 @@ def open_virtual_file(self, family, geometry, direction, data):
             if status != 0:
                 raise GMTCLibError(f"Failed to close virtual file '{vfname}'.")
 
+    def open_virtual_file(self, family, geometry, direction, data):
+        """
+        Open a GMT virtual file to pass data to and from a module.
+
+        The method has been renamed to ``open_virtualfile` since v0.11.0 and will be
+        fully removed in v0.15.0.
+
+        Examples
+        --------
```

Comment:
Is a doctest actually needed for the old `open_virtual_file` method? Or can we just omit it?

Reply:
Just want to make sure that the old function name still works as expected. But I'm OK with removing it, since it's a very simple wrapper.

Reply:
Removed in 9fcff9c1fdc4f8b08bda3f48982cd8fa2b2302d6.